pkg/scalers/postgresql_scaler.go

```diff
@@ -225,3 +224,17 @@ func (s *postgreSQLScaler) GetMetrics(ctx context.Context, metricName string, me
 
 	return append([]external_metrics.ExternalMetricValue{}, metric), nil
 }
+
+func getFromAuthOrMeta(config *ScalerConfig, field string) (string, error) {
```

Comment:
This function should be defined in some other place outside of this scaler and referenced then.

Reply:
You are right, where should I define it? Maybe in nex file? At scaler_handler level?

Reply:
🤷‍♂️ in `scaler.go` or maybe create utils package in `pkg/scalers` ?